code/part-one/README.md

```diff
@@ -147,11 +147,13 @@ best bet is probably to familiarize yourself with Node's crypto module, in
 particular using it to create SHA-512 hashes. This will come up later when you
 start working with Hyperledger Sawtooth.
 
-Now that you understand hashing, a blockchain should actually be actually
-fairly straightforward. You take some data, you gather it into a "block", you
-combine it with the hash of a previous block, and then you create a new hash.
-Now, when more data comes in, bundle it into a new block, combine it with the
-hash from your first block, and generate a new hash. Rinse and repeat.
+Now that you understand hashing, a blockchain should actually be rather
+straightforward. It's just bundles of data linked sequentially by hashes of
+that data. Start with a "genesis block". This is the only block which won't linked
```

Comment:
**(minor) Reorder punctuation:** Start with a "genesis block`."`

**Add missing word:** This is the only block which won't `be` linked
or
**Replace word:** This is the only block which ~won't~ `isn't` linked